### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by AnyoneClown**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,3 +1,45 @@
...
+            self.km += other
+        return self
+
+    def __mul__(self, other: int | float) -> Distance:
+        return Distance(self.km * other)
+
+    def __truediv__(self, other: int | float) -> Distance:
+        return Distance(round((self.km / other), 2))
+
+    def __lt__(self, other: int | Distance) -> True | False:
```
Comment (viktoria-rybenchuk) [inline app/main.py:32]:
```suggestion
    def __lt__(self, other: int | Distance) ->bool:
```

Comment (viktoria-rybenchuk) [reply app/main.py:32]: Fix in all places

File: app/main.py [hunk 2]
```diff
@@ -1,3 +1,45 @@
...
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance | int | float) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        return Distance(self.km + other)
+
+    def __iadd__(self, other: Distance | int) -> Distance:
```
Comment (viktoria-rybenchuk) [inline app/main.py:19]: `other` could be float as well

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]

**Review by Dimosphen1 — APPROVED** [review]
LGTM, well done!

**Pull request closed without merging**